OrbitCore/TracepointEventBuffer.cpp

```diff
@@ -18,15 +19,16 @@ void TracepointEventBuffer::AddTracepointEventAndMapToThreads(uint64_t time,
   event.set_tid(thread_id);
```

Comment:
`TracepointEventInfo` also has a `pid` field, but it's not set. It should probably be set as events are collected system wide and we will probably be interested in the `pid` as well (which is not always the target pid).

Reply:
Thanks for the suggestion! Will update the code.